pkg/apiserver/apis/packages/types.go

```diff
@@ -38,8 +38,17 @@ type PackageSpec struct {
 	PublicName string `json:"publicName,omitempty"`
 	Version    string `json:"version,omitempty"`
 
-	DisplayName string `json:"displayName,omitempty"`
-	Description string `json:"description,omitempty"`
+	DisplayName          string `json:"displayName,omitempty"`
+	LongDescription      string `json:"longDescription,omitempty"`
+	ShortDescription     string `json:"shortDescription,omitempty"`
+	Icon                 string `json:"icon,omitempty"`
```

Comment:
Does this need to be an svg? Maybe just base64Icon?

Reply:
if it's not svg it would be hard to incorporate it into various environments (light mode, dark mode, different sizes, etc.). separately i dont think it's good to use "base64" prefix instead of suffix since the main subject here is "icon".